frontend/src/views/WheresMyGeneV2/components/HeatMap/connect.ts

```diff
@@ -211,6 +211,10 @@ export function useConnect({
 
   const handleExpandCollapse = useCallback(
     (tissueId: string, tissueName: Tissue) => {
+      /**
+       * WARNING: Doesn't pass React StrictMode check
+       * We get double events in dev mode, will need further investigation
+       */
       dispatch?.(toggleExpandedTissueId({ tissueId, tissueName }));
```

Comment:
something we need to keep an eye on, since this code is not compatible with React StrictMode and future versions of React